src/pyflexplot/plotting/coord_trans.py

```diff
@@ -133,20 +133,24 @@ def axes_to_map(self, x, y):
         """Transform from axes to map coordinates."""
         # pylint: disable=E0633  # unpacking-non-sequence
         x_geo, y_geo = self.axes_to_geo(x, y)
-        x, y = self.geo_to_map(x_geo, y_geo)
+        x_map, y_map = self.geo_to_map(x_geo, y_geo)
 
         if (
             isinstance(self.proj_map, PlateCarree)
-            and not isinstance(x, float)
+            and not isinstance(x_map, float)
             and len(x) == 2
-            and np.isclose(x[0], x[1], atol=0.5)
-            and np.isclose(np.abs(x[0]), 180, atol=0.5)
         ):
-            # Fix edge case where lon range (-180, 180) is turned into (-180, -180)
+            # Fix edge case where lon range (-180, 180) can invert to (180, -180)
```

Comment:
Are there any test cases that expose this behaviour?

Reply:
Yes, there is one test which covers this edge case